dissect/target/loaders/tar.py

```diff
@@ -42,7 +42,8 @@ def map(self, target: target.Target) -> None:
             if member.name == ".":
                 continue
 
-            if not member.name.startswith("fs/") and not member.name.startswith("/sysvol"):
+            sysvols = ["/fs", "fs/", "/sysvol", "sysvol/"]
+            if not any([member.name.startswith(s) for s in sysvols]):
```

Comment:
```suggestion
            if not member.name.startswith(("/fs", "fs/", "/sysvol", "sysvol/")):
```
`startswith` also accepts a tuple of prefixes to check.